c/secp256k1_blake160_sighash_all.c

```diff
@@ -53,11 +53,13 @@
 // the 64-byte compact recoverable signature.
 #define BLAKE2B_BLOCK_SIZE 32
 #define BLAKE160_SIZE 20
+#define MAX_ARGS_SIZE 32
```

Comment:
it can be larger in case more args are required later.

Reply:
We decided to set the `MAX_AGRS_SIZE` to 64. cc @XuJiandong